### PR TITLE
Added script for comparing assets for 2 releases

### PR DESCRIPTION
**Pull request by IvanRuzavin**

### CODE REVIEW

File: .github/workflows/compareAssets.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            ${{ runner.os }}-pip-
+
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install requests
+          pip install chardet
+          pip install py7zr
+
+      - name: Run Keyword Replacement Script
```
Comment (StrahinjaJacimovic) [inline .github/workflows/compareAssets.yaml:42]: Name it accordingly.

File: scripts/compare_assets.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    for previous_asset in previous_assets:
+        if previous_asset not in current_assets:
+            print(f"\033[93m{previous_release['tag_name']} release has {previous_asset} which is not present in {current_release['tag_name']}!")
+            found_dif = 1
+
+    if found_dif == 0:
+        print(f"\033[92mThere is no difference in asset names between {current_release['tag_name']} and {previous_release['tag_name']} releases!")
+
+    return
```
Comment (StrahinjaJacimovic) [inline scripts/compare_assets.py:69]: Before returning, write all diffs to a file and add as artifact to pipe run.

File: .github/workflows/compareAssets.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          type: string
+          description: First Release Tag
+          default: "latest"
+        second_tag:
+          type: string
+          description: Second Release Tag (latest stays for tag before latest)
+          default: "latest"
+
+jobs:
+  upload-release-asset:
```
Comment (StrahinjaJacimovic) [inline .github/workflows/compareAssets.yaml:16]: Don't run it if both values are left as latest.

File: scripts/compare_assets.py (hunk 2 above)
Comment (IvanRuzavin) [reply scripts/compare_assets.py:75]: Done.

File: scripts/compare_assets.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    current_assets = [package.replace('.7z', '') for package in fetch_existing_asset_names(current_release)]
+    previous_assets = [package.replace('.7z', '') for package in fetch_existing_asset_names(previous_release)]
+
+    for current_asset in current_assets:
+        if current_asset not in previous_assets:
+            print(f"\033[93m{current_release['tag_name']} release has {current_asset} which is not present in {previous_release['tag_name']}!")
+            # Write the line to a file
+            with open("result_compare.txt", "a") as file:
+                file.write(f'{current_release['tag_name']} release has {current_asset} which is not present in {previous_release['tag_name']}!\n')
+            found_dif = 1
```
Comment (StrahinjaJacimovic) [inline scripts/compare_assets.py:55]: Change `found_dif` to bool approach.

**Review by StrahinjaJacimovic — CHANGES_REQUESTED** [review]

File: .github/workflows/compareAssets.yaml (hunk 3 above)
Comment (StrahinjaJacimovic) [reply .github/workflows/compareAssets.yaml]: Change it to an IF clause.

i.e.

```
if: ${{ selection1 != selection2 }}
```

**Review by StrahinjaJacimovic — APPROVED** [review]

**Pull request merged**